src/js/components/AppVersionListComponent.jsx

```diff
@@ -163,17 +163,23 @@ var AppVersionListComponent = React.createClass({
   },
 
   render: function () {
+    var currentVersion = AppsStore.getCurrentApp(this.props.appId);
+
+    var versionDate = currentVersion.version != null ?
+      ` - ${new Date(currentVersion.version).toLocaleString()}` :
```

Comment:
This is only called if the condition will match, so bringing this into an own variable would look like this (can not use the ternary then) :
```
var versionDate = "";
if (currentVersion != null) {
  let currentVersionLocale = new Date(currentVersion.version).toLocaleString();
  versionDate = ` - ${currentVersionLocale}`;
}
```
I like both variants, do you still prefer the second? Then I will stick with that.